packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -19,26 +19,62 @@ import {
   Tab,
   TabPanel,
 } from '../components';
-import { DEFAULT_DASHBOARD_GROUP } from '../constants';
+import { DEFAULT_DASHBOARD_GROUP, MIN_DATA_YEAR } from '../constants';
 
 const DashboardSection = styled(FlexCenter)`
   min-height: 31rem;
 `;
 
+const StyledSelect = styled(Select)`
+  margin: 0 1rem 0 0;
+  width: 8rem;
+`;
+
+const getYearOptions = () => {
+  const currentYear = new Date().getFullYear();
+  const years = [
+    {
+      label: 'All Data',
+      value: 'all',
+    },
+  ];
+  let year = MIN_DATA_YEAR;
```

Comment:
Should `MIN_DATA_YEAR` be a number rather than a string, given the way it's used here? Clearly it must work, but it feels a bit type-loose!

Reply:
Yes that's a good point - incrementing on the string is a bit dodgy. Love the heightened type awareness hangover from typescript.